doc/ref_cert/RC1/chapters/chapter02.rst

```diff
@@ -962,49 +952,115 @@ This optimization should be based on the following principles:
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
-| **MVP     | vnf.nfvi.\  | 50\     | 20\     | 20\   | 10\   | 40\   | 40\   |
-| Metrics** | perf.001    | Gbps    | Gbps    | Gbps  | Gbps  | Gbps  | Gbps  |
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
-|           | vnf.nfvi.\  | n/a (4) | n/a (4) | ?     | ?     | ?     | 0.5ms |
-|           | perf.002    |         |         |       |       |       |       |
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
-|           | infra.\     | n/a (1) | n/a (1) | 40\   | 20\   | 60\   | 80\   |
-|           | nfvi.\      |         |         | Gbps  | Gbps  | Gbps  | Gbps  |
-|           | perf.001    |         |         |       |       |       |       |
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
-|           | infra.\     | n/a (1) | n/a (1) | ?     | ?     | ?     | ?     |
-|           | nfvi.\      |         |         | W/\   | W/\   | W/\   | W/\   |
-|           | perf.007    |         |         | Gbps  | Gbps  | Gbps  | Gbps  |
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
-|           |             |         |         |       |       |       |       |
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
-| **Non-MVP | vnf.nfvi.\  | n/a (4) | n/a (4) | ?     | ?     | ?     | 1ms   |
-| Metrics** | perf.003    |         |         |       |       |       |       |
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
-|           | vnf.nfvi.\  | n/a (4) | n/a (4) | ?     | ?     | ?     | 500k  |
-|           | perf.004    |         |         |       |       |       |       |
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
-|           | vnf.nfvi.\  | n/a (4) | n/a (4) | ?     | ?     | ?     | 110\  |
-|           | perf.005    |         |         |       |       |       | kfps  |
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
-|           | infra.\     | n/a (1) | n/a (1) | ?     | ?     | ?     | 1G    |
-|           | nfvi.\      |         |         |       |       |       |       |
-|           | perf.002    |         |         |       |       |       |       |
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
-|           | infra.\     | n/a     | n/a     | ?     | ?     | ?     | 200\  |
-|           | nfvi.\      | (1)     | (1)     |       |       |       | kfps  |
-|           | perf.003    |         |         |       |       |       |       |
-+-----------+-------------+---------+---------+-------+-------+-------+-------+
+.. list-table:: Optimized matrix model of the test cases against the metrics
+                to be measured
+   :widths: 20 20 10 10 10 10 10 10
+   :header-rows: 1
+
+   * -
+     - NFVI Profiles
+     - B
+     -
+     -
+     -
+     - N
```

Comment:
should this be "H" for HIgh Performance? Maybe we can use full rather than abbreviation as the abbreviation is not defined.
```suggestion
     - H
```

Reply:
I will check. It's out of the scope of this PR. thx